README.md

```diff
@@ -59,43 +83,40 @@ The canonical download instructions for IPFS are over at: https://docs.ipfs.io/g
 
 ### System Requirements
 
-IPFS can run on most Linux, macOS, and Windows systems. We recommend running it on a machine with at least 2 GB of RAM and 2 CPU cores (go-ipfs is highly parallel). On systems with less memory, it may not be completely stable.
+IPFS can run on most Linux, MacOS, and Windows systems. We recommend running it on a machine with at least 2 GB of RAM and 2 CPU cores (go-ipfs is highly parallel). On systems with less memory, it may not be completely stable.
```

Comment:
macOS is, unfortunately, the "correct" capitalization (according to apple).

Reply:
Ah okay :D